presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1335,7 +1335,7 @@ public void testArrays()
         assertQuery("SELECT col[2] FROM tmp_array1", "SELECT 2");
         assertQuery("SELECT col[3] FROM tmp_array1", "SELECT NULL");
 
-        assertUpdate("CREATE TABLE tmp_array2 AS SELECT ARRAY[1.0, 2.5, 3.5] AS col", 1);
+        assertUpdate("CREATE TABLE tmp_array2 AS SELECT ARRAY[DOUBLE '1.0', DOUBLE '2.5', DOUBLE '3.5'] AS col", 1);
```

Comment:
Instead of changing these to DOUBLE 'xxx' we should consider changing the rendering of doubles to what the SQL specifies for approximate numerics (i.e., scientific notation)